src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -52,6 +54,8 @@ public data class YamlConfiguration(
     internal val allowAnchorsAndAliases: Boolean = false,
     internal val yamlNamingStrategy: YamlNamingStrategy? = null,
     internal val codePointLimit: Int? = null,
+    @ExperimentalSerializationApi
```

Comment:
note: From my understanding, the idea of experimental annotations is that each project has its own annotations that can be used to mark experimental parts of the API. If this part of the API is experimental then, probably, you should create an experimental annotation in this repo and use in instead of the one declared in kotlinx-serialization library

Reply:
As of the `kaml` project is a extension of the `kotlinx.serialization` project. It would be more appropriate to use their experimental annotations. It will also make the code consistent across other kotlinx serializers.

Reply:
Got it. Thank you for clarification. Also checked some other `kotlinx.serialization` libraries - they use the same approach. For example, [ktoml](https://github.com/orchestr7/ktoml/blob/main/ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/decoders/TomlArrayDecoder.kt#L20)